bat-utils/lib/hapi-server.js

```diff
@@ -41,13 +47,14 @@ function pushTokens (map) {
   }
 }
 
-const Server = async (options, runtime) => {
+async function Server (options, runtime) {
   const debug = new SDebug('web')
 
-  const graylist = { addresses: process.env.IP_GRAYLIST && process.env.IP_GRAYLIST.split(',') }
-  const server = new hapi.Server()
-
-  server.connection({ port: process.env.PORT })
+  const serverOpts = {
+    port: options.port,
+    host: '0.0.0.0'
```

Comment:
why 0.0.0.0 instead of 127.0.0.1?

Reply:
i believe this was to work with docker containers

```The host property set to localhost is likely the safest choice. In a docker container, however, the localhost may not be accessible outside of the container and using host: '0.0.0.0' may be needed.
```
https://hapi.dev/tutorials/

just ran the tests with this change locally, got a connection refused error